package.json

```
@@ -21,11 +21,11 @@
   "devDependencies": {
     "husky": "^4.2.3",
     "lint-staged": "^10.0.8",
-    "pre-commit": "^1.2.2",
+    "pre-commit": "^1.0.10",
```

Why is the pre-commit dependency reverted to an older version here?

---

fair! we should keep it to the current, I think that was me trying to mitigate local pre-commit errors that arose

```suggestion
    "pre-commit": "^1.2.2",
```